### ISSUE TITLE
bug with the barrier when getting combat tagged

### ISSUE THREAD

**Issue by Twi5TeD** [MinelinkNetwork/CombatTagPlus#88]
if you get combat tagged and then right click the glass it will disappear and you can walk into the safe zone and the tag will no longer be active

**Comment by Techcable** [comment]
@Byteflux 
We might want to consider switching to sending updates every tick instead of on move, like I did in [SpawnShield](https://github.com/TechzoneMC/SpawnShield/blob/master/base/src/main/java/net/techcable/spawnshield/tasks/ForceFieldUpdateTask.java).
You may also want to consider caching the location of the border-blocks in order to reduce update delays.
Actually, precomputing the border blocks on the main thread may actually help simplify the plugin and avoid having to handle concurrency with the other plugins.

**Comment by Twi5TeD** [comment]
any news on this?

**Comment by Byteflux** [comment]
The glass barrier is a visual thing only, it's not the actual forcefield that blocks people from passing through. People being able to break the glass doesn't make them able to walk through, if it does, it's probably a different plugin interfering somehow. Make sure you are using the latest version, because I can't reproduce this bug.

**Comment by Twi5TeD** [comment]
Yes I'm using the latest version not sure what else could cause it though :/ looks like  will have to disable each plugin 1 by 1 to find out if it's another plugin that's causing it

**Comment by Twi5TeD** [comment]
hmm it seems to be my config that causes it as the default one worked i changed my settings to this here is a link  could you tell me whats wrong with it thanks http://pastebin.com/9MSxdXY1

**Comment by Twi5TeD** [comment]
any news? its my actual config that causes it if you try it yourself with my config
